#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("downsample_rate", double_t, 0, "point cloud downsampling rate", 0.5, 0.0001, 1.0)
gen.add("sphere_radius", double_t, 0, "radius of sphere", 1, 0.0, 100.0)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "FisheyeSphere"))
